doc/nrf/ug_tfm.rst

```diff
@@ -104,7 +107,27 @@ The logs arrive on different COM ports on the host PC.
    * On the nRF5340 DK v2.0.0, there is one fewer COM port than on v1.0.0, so the secure and non-secure UART peripheral must be wired to the same pins.
      Specifically, wire the pins **P0.25** and **P0.26** to **P0.20** and **P0.22**, respectively.
 
-Limitations
-***********
+Migrating from Secure Partition Manager to Trusted Firmware-M
+*************************************************************
+
+The interface to TF-M is different from the interface to SPM.
+Due to that, the application code that uses the SPM :ref:`lib_secure_services` needs to be ported to use TF-M instead.
+
+The following SPM services can be replaced by TF-M:
 
-Application code that uses SPM :ref:`lib_secure_services` cannot use TF-M because the interface to TF-M is different and, at this time, not all SPM functions are available in TF-M.
+* ``spm_request_system_reboot`` uses ``tfm_platform_system_reset``.
+* ``spm_request_random_number`` uses ``psa_generate_random`` or ``entropy_get_entropy``.
+* ``spm_request_read`` uses ``tfm_platform_mem_read`` or ``soc_secure_mem_read``
+* ``spm_s0_active`` uses ``tfm_platform_s0_active``
+* ``spm_firmware_info`` uses ``tfm_firmware_info``
+
+The following SPM services have no replacement in TF-M:
+
+* ``spm_prevalidate_b1_upgrade``
+* ``spm_busy_wait``
+* ``spm_set_ns_fatal_error_handler``
+
+.. note::
+   By default, TF-M configures memory regions as secure memory, while SPM configures memory regions as non-secure by default.
```

Comment:
When I think about it, this 'by default' in the end of this sentence can be removed.